content/posts/redis-modules-into-arm-land-part1.md

```diff
@@ -0,0 +1,276 @@
+---
+title: "Getting Redis Modules Into ARM Land - Part 1"
+date: 2019-11-13
+authors:
+  - author:
+      name: "Rafi Einstein"
+      link: "https://github.com/rafie"
+tags: ["redisedge", "arm"]
+---
+
+Here at Redis Labs, RedisEdge was our motivation for getting into ARM land and dragging Redis modules along.

```

Comment:
Sorry, this sounds weird to me as I read it back now, so let's change to "Here at Redis Labs, our motivation to bring Redis modules into ARM land was RedisEdge."